### PR TITLE
Add Russian docs into 8.x sections

### PR DESCRIPTION
**Pull request by likhobory**

### CODE REVIEW

**Comment by samus-aran** [comment]
Ack A lot of unwanted commits in the PR :sweat_smile: are you able to update this and then I'll have a review ASAP.

**Comment by likhobory** [comment]
Ashley, pay attention that this is a Russian branch, usually @pgorod updated it before merging, or do you want to merge  it directly into master ?

**Comment by samus-aran** [comment]
Ack @likhobory You are absolutely right :facepalm: I'll get that done

File: content/8.x/admin/installation-guide/Upgrading.ru.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ {{% /notice %}}
+
+
+== Обновление системы
+
+В данном разделе описывается процесс обновления системы при помощи мастера обновления. 
+
+Прежде всего, необходимо скачать необходимый https://suitecrm.com/upgrade-suitecrm[пакет обновления^].
+ 
+{{% notice warning %}}
```
Comment (samus-aran) [inline content/8.x/admin/installation-guide/Upgrading.ru.adoc:35]: Perhaps just a note for myself to update the english version to include this visual warning

File: content/8.x/admin/installation-guide/Upgrading.ru.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+:imagesdir: /images/ru/admin/Upgrading
+
+ifdef::env-github[:imagesdir: ./../../../../master/static/images/ru/admin/Upgrading]
+
+:btn: btn:
+
+ifdef::env-github[:btn:]  
+
+{{% notice note %}}
```
Comment (samus-aran) [inline content/8.x/admin/installation-guide/Upgrading.ru.adoc:24]: Better to have info or note?

Comment (likhobory) [reply content/8.x/admin/installation-guide/Upgrading.ru.adoc:24]: Depends on the context, but in this case I considered this message to be quite important, I mark less important messages as info. IMHO.

**Pull request merged**